packages/terra-tabs/src/Tabs.jsx

```diff
@@ -119,16 +135,40 @@ class Tabs extends React.Component {
           showIcon={child.props.showIcon}
           render={() => tabContent}
           isDisabled={child.props.isDisabled}
+          isClosable={this.props.isClosable !== undefined ? this.props.isClosable : false}
```

Comment:
Set the default value for `isClosable` as `false` and spread the prop  along with other props in above lines.

```suggestion
          isClosable={isClosabale}
```

Reply:
Done , https://github.com/cerner/terra-framework/commit/c007c2b50613c2908c4aab55530060a38ea67584